docs/appendices/release-notes/unreleased.rst

```diff
@@ -55,6 +55,9 @@ Deprecations
 Changes
 =======
 
+- Added the :ref:`pg_postmaster_start_time() <scalar_pg_postmaster_start_time>`
+  scalar function.
```

Comment:
```suggestion
  :ref:`scalar function <scalar-functions>`.
```

Reply:
Sorry, mergify already merged before I saw this. Do you have a strong opinion on this? I tend to prefer only one link in this case as it's the same document anway and the users don't have insights into where the links are pointing to - so they might be inclined to click on both and discover that it's the same thing anyways